bylaws.md

```diff
@@ -0,0 +1,118 @@
+
+### D. Curricula
+The Academie d’Espee Council will provide a minimum Rapier 101 Core Curriculum that every student of the art of defense is encouraged to learn over the course of their early studies and deepen their understanding of throughout their fencing career. Classes in these topics will be common at AAoF events and local practices. Advanced curricula lists may be submitted to the Information Officer for the Ad’E website for publication, allowing for a clearing house of other curricula. Areas of particular interest include a curriculum for instructors and recommended reading lists.
+
+
+### V. Process for amending the bylaws
+
+Proposals to change these bylaws may be made by any member of the Academie. They are voted on by the members of the Academie and adopted by majority vote of those voting. All change proposals are accumulated and voted upon simultaneously in an annual polling that occurs in July with an optional polling day in January at the discretion of the council. The process is detailed below.
+
+<ol type="A">
+<li>Informal discussion: Ideas for changes can take place at any time and in any forum. It is recommended that public notification of the discussion be made on the Academie’s chosen social media channels.</li>
+<li>Draft Proposal: A draft of the change must be submitted to the Bylaw Clerk no later than 2 months prior to the polling. If the proposal is a change from existing policy it will specify what text is being replaced (“from”) and what the new text will be (“to”). New sections may also be proposed.</li>
+<li>Proposal editing: The Bylaw Clerk and the Executive Council may suggest wording changes to a proposal with the final wording still up to the proposer within reason. In the event of conflicting proposals (e.g. one that changes A to B and one that changes A to C) the executive council may adjust the structure of the polling elements to ensure a sane result (e.g. not both B and C). In this case, the polling would likely be: pick A, B, or C with the choice receiving the most votes winning.</li>
+<li>Final proposal text: The final proposal text will be locked at 1 month prior to the polling and will be published and announced for membership consideration. Notice will be made on the Ad’E social media channels with short text proposals also being posted there or links provided to the location of longer proposals.</li>
+<li>Polling: All proposals will be voted upon simultaneously via an electronic ballot with the polling open for one week open to all Academie members. If a majority of the votes cast support the change, it will be adopted. “Yes” votes must exceed “No” votes with “Abstain” votes having no effect on the result. A tie results in failure of the proposal.</li>
+<li>Updating: The Bylaw Clerk will report on the polling results and update the official posted bylaws to reflect all approved changes. Changes will be in effect one month after the closing of the poll that approved them.</li>
+</ol>
```

Comment:
```suggestion

1. Informal discussion: Ideas for changes can take place at any time and in any forum. It is recommended that public notification of the discussion be made on the Academie’s chosen social media channels.
1. Draft Proposal: A draft of the change must be submitted to the Bylaw Clerk no later than 2 months prior to the polling. If the proposal is a change from existing policy it will specify what text is being replaced (“from”) and what the new text will be (“to”). New sections may also be proposed.
1. Proposal editing: The Bylaw Clerk and the Executive Council may suggest wording changes to a proposal with the final wording still up to the proposer within reason. In the event of conflicting proposals (e.g. one that changes A to B and one that changes A to C) the executive council may adjust the structure of the polling elements to ensure a sane result (e.g. not both B and C). In this case, the polling would likely be: pick A, B, or C with the choice receiving the most votes winning.
1. Final proposal text: The final proposal text will be locked at 1 month prior to the polling and will be published and announced for membership consideration. Notice will be made on the Ad’E social media channels with short text proposals also being posted there or links provided to the location of longer proposals.
1. Polling: All proposals will be voted upon simultaneously via an electronic ballot with the polling open for one week open to all Academie members. If a majority of the votes cast support the change, it will be adopted. “Yes” votes must exceed “No” votes with “Abstain” votes having no effect on the result. A tie results in failure of the proposal.
1. Updating: The Bylaw Clerk will report on the polling results and update the official posted bylaws to reflect all approved changes. Changes will be in effect one month after the closing of the poll that approved them.
```